.github/workflows/publish.yml

```diff
@@ -1,9 +1,15 @@
 name: Publish Extension
 
-on: workflow_dispatch
+on:
+  pull_request:
+    types: [closed]
 
 jobs:
   deploy:
+    if: |
+      github.event.pull_request.merged == true &&
+      contains(github.event.pull_request.labels.*.name, 'make release go now') && 
+      github.event.pull_request.title == 'Update version and CHANGELOG for release'
```

Comment:
if a PR is closed and merged and the labels contain the `make release go now` label and the title is 'Update version and CHANGELOG for release'